charts/kvisor/values.yaml

```diff
@@ -156,17 +156,10 @@ controller:
 #    capabilities:
 #      drop: [ ALL ]
 
-  resources: { }
-    # We usually recommend not to specify default resources and to leave this as a conscious
-    # choice for the user. This also increases chances charts run on environments with little
-    # resources, such as Minikube. If you do want to specify resources, uncomment the following
-    # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
-    # limits:
-    #   cpu: 100m
-    #   memory: 128Mi
-    # requests:
-  #   cpu: 100m
-  #   memory: 128Mi
+  resources:
+    requests:
+      cpu: 100m
+      memory: 256Mi
```

Comment:
Wait, I am confused, didn't we say we stick with limits for memory?

Reply:
no this is `requests`.

Reply:
I know that those are the requests 😅 I thought we still agreed to set memory limit = request.